fs/io_zenfs.cc

```diff
@@ -122,6 +126,12 @@ Status ZoneFile::DecodeFrom(Slice* input) {
         extent->zone_->used_capacity_ += extent->length_;
         extents_.push_back(extent);
         break;
+      case kCTime:
+        uint64_t ct;
+        if (!GetFixed64(input, &ct))
+          return Status::Corruption("ZoneFile", "Missing creation time");
```

Comment:
This is breaking backwards compatibility!! so previously created file system cant be mounted!